src/screens/PostPageDetail/index.js

```diff
@@ -41,7 +40,6 @@ const FeedsPostDetail = (props) => {
         feedId={feedId}
         dispatch={dispatch}
         setFeedByIndexProps={setFeedByIndex}
-        navigateToReplyView={navigateToReplyView}
         page={props.route.name}
         contextSource={contextSource || CONTEXT_SOURCE.FEEDS}
         haveSeeMore={haveSeeMore}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `navigateToReplyView` prop has been removed from the component. Ensure that this change doesn't affect any functionality where this prop was used in the child components. If the function is no longer needed, also ensure it's removed from its definition place to maintain clean code.
```diff
-        navigateToReplyView={navigateToReplyView}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->